db/db_test.go

```diff
@@ -0,0 +1,25 @@
+package db
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestBuildDataSourceName(t *testing.T) {
+	assert := assert.New(t)
```

Comment:
That's how they did it in examples, I think you need to pass in the test object so that it can halt operation if an assertion fails.